vms/avm/states/state.go

```diff
@@ -419,22 +422,10 @@ func (s *state) GetStatus(id ids.ID) (choices.Status, error) {
 		return *status, nil
 	}
 
-	val, err := database.GetUInt32(s.statusDB, id[:])
-	if err == database.ErrNotFound {
-		s.statusCache.Put(id, nil)
-		return choices.Unknown, database.ErrNotFound
-	}
-	if err != nil {
-		return choices.Unknown, err
-	}
-
-	status := choices.Status(val)
-	if err := status.Valid(); err != nil {
-		return choices.Unknown, err
+	if tx, err := s.GetTx(id); tx != nil && err == nil {
+		return choices.Accepted, nil
 	}
-
-	s.statusCache.Put(id, &status)
-	return status, nil
```

Comment:
I think for this PR we should leave the `GetStatus` function unmodified because during the syncing of the DAG txs are still being persisted with their processing status.